### DISCUSSION TITLE
I have 2 very important questions that I can't quite find answers to

### DISCUSSION THREAD

**Discussion by TuckerAndrews3** [eccentricdevotion/TARDIS#672] [Q&A] [answered]
1. How do we use the Telepathic circuit? We've got it placed in our TARDISes, pref set to on. Where do we go from there? Is there any perm we need to give?

2. What does the perm "tardis.display" do?

**Comment by eccentricdevotion** [comment, marked as answer]
### Telepathic circuit

Companions should right-click the telepathic circuit, then they can run `/tardis` or `/tardistravel` commands that will affect the TARDIS they are in (rather than their own TARDIS if they have one). The only permission is `      tardis.prefs.telepathy` which allows a timelord to change their telepathic preferences.

### Display

`tardis.display` lets players use the TARDIS HUD. Activate the HUD with the command `/tardisdisplay [ALL|BIOME|COORDS|DIRECTION|TARGET_BLOCK]`. There is one config option which uses colour codes and placeholders to show information:

```
# HUD display
display:
  all: '&6X&7%X% &6Y&7%Y% &6Z&7%Z% &6F&7%FACING% (%FACING_XZ%) %TARGET_BLOCK%'
```
   
The other HUD display options use the following (not configurable):

BIOME: `"&6B&7%BIOME%"`
COORDS: `"&6X&7%X% &6Y&7%Y% &6Z&7%Z%"`
DIRECTION: `"&6F&7%FACING% (%FACING_XZ%)"`
TARGET_BLOCK: `"&6T&7%TARGET_BLOCK%"`

![HUD](https://user-images.githubusercontent.com/1841924/200494646-983b72d2-db51-4c3b-a015-4f4e93b4723f.png)

**Comment by TuckerAndrews3** [comment]
Thank you so much!
